rockPaperScissors.py

```diff
@@ -3,48 +3,51 @@
 
 import random
 
-def gameLogic(player, computer, playerScore, computerScore):
-    playerIndex = RPS.index(player)
-    computerIndex = RPS.index(computer)
-
-    gameIncrement = 0
-    if (playerIndex != computerIndex):
-        gameIncrement = 1
-        playerNextIndex = (playerIndex + 1) % len(RPS)
-        
-        if (playerNextIndex == computerIndex):
-            computerScore += 1
-        else:
-            playerScore += 1
+def gameLogic(player1Choice, player2Choice):
+    if player1Choice == player2Choice:
+        return 0
 
-    return gameIncrement, playerScore, computerScore
+    playerNextIndex = (player1Choice + 1) % 3
+    
+    winner = 2 if playerNextIndex == player2Choice else 1
```

Comment:
You may or may not have seen ternary statements yet. They're weird at first, but really great if  you take the time to get used to them. Fortunately, python's ternary syntax is way less...weird...than most other languages. Basically, if the thing in the middle is true, you'll get the value on the left side. If it's false, you'll get the value on the right side. in this case, if ```playerNextIndex == player2Choice```, ```winner``` will equal 2. otherwise it'll be 1. Python makes that almost human readable: Give me 2 if playerNextIndex == player2Choice, otherwise give me 1

Reply:
I have not seen this before. research will be done